2023-0X-XX-qubes-os-4-2-0-rc1-available-for-testing.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Qubes OS 4.2.0-rc1 is available for testing"
+categories: releases
+download_url: /downloads/
+---
+
+We're pleased to announce that the first [release candidate](#what-is-a-release-candidate) for Qubes OS 4.2.0 is now available for [testing](/doc/testing/). This [minor release](#what-is-a-minor-release) includes several new features and improvements over Qubes OS 4.1.0. Qubes 4.2.0-rc1 is available on the [downloads](/downloads/) page.
+
+
+## What's new in Qubes 4.2.0?
+
+- Dom0 upgraded to Fedora 37
+- Xen updated to version 4.17
+- SELinux support in Fedora templates
+- Several GUI applications rewritten, including:
+  - Applications Menu
+  - Qubes Global Settings
+  - Create New Qube
+  - Qubes Update
+- Unified `grub.cfg` location for both UEFI and legacy boot
+- PipeWire support
+- fwupd integration for firmware updates
+- Optional automatic clipboard clearing
+- Official packages built using Qubes Builder v2
+
+Please see the [Qubes OS 4.2.0 release notes](/doc/releases/4.2/release-notes/) for details.
+
+## Testing Qubes 4.2.0-rc1
+
+If you're willing to [test](/doc/testing/) this release candidate, you can help us improve the eventual stable release by [reporting any bugs you encounter](/doc/issue-tracking/). We encourage experienced users to join the [testing team](https://forum.qubes-os.org/t/joining-the-testing-team/5190).
+
+A full list of known bugs in Qubes 4.2.0 is available [here](https://github.com/QubesOS/qubes-issues/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Release+4.2%22+label%3A%22T%3A+bug%22). We strongly recommend [updating Qubes OS](/doc/how-to-update/) immediately after installation in order to apply all available bug fixes.
+
+## Upgrading to Qubes 4.2.0-rc1
+
+It is not yet possible to perform an in-place upgrade from Qubes 4.1 to Qubes 4.2. For this initial release candidate, a clean installation is required. An in-place upgrade tool is in development.
+
+## When is the stable release?
+
+That depends on the number of bugs discovered in this release candidate and their severity. Our usual process after issuing a new release candidate is to collect bug reports, triage the bugs, and fix them. This can take anywhere from a week to over a month, but it's usually a few weeks. We then issue a new release candidate that includes the fixes, let folks test it, and repeat the process until we're left with a release candidate that's good enough to be declared the stable release. No one can predict, at the outset, how many iterations of this process will be required (and hence how many release candidates will be needed before a stable release), but we generally have a sense when we're getting close, and we'll indicate that in this section in future release candidate announcements. In the event that no severe bugs are discovered in 4.2.0-rc1, we expect to announce Qubes 4.2.0-rc2 in approximately three weeks.
```

Comment:
It's 5 weeks between rc-s: https://www.qubes-os.org/doc/version-scheme/#release-schedule
And there definitelly will be rc2 (if nothing else, then the in-place upgrade will need testing). The last sentence I think was for the final release originally (3 weeks after rc), but -rc1 surely won't be the final one.

Reply:
> It's 5 weeks between rc-s: https://www.qubes-os.org/doc/version-scheme/#release-schedule

Oh, I must've forgotten. Thanks for pointing that out. Will fix text.

> And there definitelly will be rc2 (if nothing else, then the in-place upgrade will need testing). The last sentence I think was for the final release originally (3 weeks after rc), but -rc1 surely won't be the final one.

Right, I figured as much, which is why the last sentence here says we *will* release an rc2. Is it easy to miss the `-rc2` at the end? I'll reword to something like "the first/second release candidate" to avoid this.

Reply:
> https://www.qubes-os.org/doc/version-scheme/#release-schedule

Is it really wise to say that it's "five weeks" between RCs and that "there should be no less than 1 and no more than 3 release candidates before final release" when, in practice, these rules are often broken? (For example, 4.1 has four RCs, 4.0 had five, etc.) I feel like this is unnecessarily setting users up for disappointment, whereas just being realistic would avoid this. At the very least, I suggest we adjust the wording to say that we *aim* for these time frames but that actual times will vary.

Reply:
> I haven't missed the `-rc2`, but the 3 weeks and the "no severe bugs" would match the final release not the next rc.

Hmm, okay.

> The "no more than 3" surely can be dropped, in practice it very much depends on the number of issues. We kinda hints in the rc announcements how many more we expect.

Okay, sounds good. Just updated the text here, and I'll also update that doc page.